docs/config-flatcar-v1_0.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Flatcar v1.0.0
+parent: Configuration specifications
+nav_order: 99
```

Comment:
To maintain the correct sort order, let's add 50 to the `nav_order` of every `rhcos` and `openshift` spec doc (in a separate commit).